AnnoyancesFilter/Cookies/sections/cookies_general.txt

```diff
@@ -2067,6 +2067,8 @@ _europecookielaw.
 ||usercentrics.eu^$third-party
 ||consensu.org^$domain=~consensu.org
 ||cmp.uniconsent.com^$third-party,domain=~xda-developers.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/181540
+||cdn.iubenda.com/cookie_solution^$domain=centrocommercialeaura.it
```

Comment:
We have common rules for this type of the cookies.
Find a rule below and add the domain to the end of it.
```adblock
||cdn.iubenda.com/cs/iubenda_cs.js
```

Reply:
I guess it will not work, because we have /cs, and they use /cookie_solution/iubenda_cs/1.60.3/core-it.js

Reply:
Hm, didn't see that on my end. Try this)
```adblock
||cdn.iubenda.com^$domain=centrocommercialeaura.it
```

Reply:
Yeah, that's good& Where should I place it? new to this `||cdn.iubenda.com/cs/iubenda_cs.js`?